examples/example_with_all.cpp

```diff
@@ -13,6 +13,8 @@ int main()
 {
     crow::SimpleApp app;
 
+    app.server_name("CrowCpp");
```

Comment:
Might be best to have this at the bottom (as part of the `app.port(1234).nultithreaded().run()` line).